annotation/tests/test_annotations.py

```diff
@@ -0,0 +1,85 @@
+import uuid
+from datetime import datetime
+
+import pytest
+from sqlalchemy import INTEGER, VARCHAR, Column, DateTime
+from sqlalchemy.dialects.postgresql import UUID
+
+from annotation.annotations.main import row_to_dict
+from annotation.database import Base
+
+
+class AnnotationRow:
+    def __init__(
+        self,
+        uuid_attr: uuid.UUID,
+        datetime_attr: datetime,
+        str_attr: str,
+        int_attr: int,
+    ):
+        self.uuid_attr = uuid_attr
+        self.datetime_attr = datetime_attr
+        self.str_attr = str_attr
+        self.int_attr = int_attr
+
+
+class AnnotationRowTable(Base):
+    __tablename__ = "test"
+
+    uuid_attr = Column(UUID(as_uuid=True), primary_key=True)
+    datetime_attr = Column(DateTime())
+    str_attr = Column(VARCHAR)
+    int_attr = Column(INTEGER)
```

Comment:
It complicates things a lot: lets focus on the function you test, abstract from other libraries. Or use existing entities we defined in annotation, instead of create new ones here.

Reply:
You could simply create a mock with __dict__ value you need

Reply:
Done